src/graphql/fragments/actions.graphql

```diff
@@ -36,6 +36,7 @@ fragment ColonyAction on ColonyAction {
       tokenAddress: id
     }
   }
+  showInActionsList
```

Comment:
The only thing I can't really wrap my head around is how this value is going to work when you have an voting rep extension installed, create a motion, it passes, then you uninstall the voting rep extension.

But that's maybe because I'm missing something

Reply:
I was imagining we'd leave it in but label it with a different a tag.